api/proto/printable.proto

```diff
@@ -45,11 +45,30 @@ message TransferPayload {
     bytes bip39_entropy = 4;
 }
 
+/// Message encoding information about a TxOut required to locate a TxOut,
+/// un-blind the amount, and spend the TxOut. This can be used to give
+/// MobileCoin to a both FOG users & non-FOG users who may not yet have
+// a MobileCoin account enabled
+message GiftCode {
```

Comment:
Yeah I changed it to gift code as I was thinking this would be our way of doing gift codes in the future, but it could be feasible that other types might arise. I'm happy to revert.

Reply:
I don't have any strong feelings about this.

Reply:
Ideally this would match the filename